contracts/always-succeeds/impl/plu-ts/make

```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+set -e
+
+npm install --silent --registry=https://npm.pkg.github.com
```

Comment:
This line fails for me. Not sure what I need to change?

Reply:
```
npm ERR! code E401
npm ERR! 401 Unauthorized - GET https://npm.pkg.github.com/@harmoniclabs%2fplu-ts - authentication token not provided

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/niels/.npm/_logs/2023-03-04T18_15_39_523Z-debug-0.log
npm ERR! code 1
npm ERR! path /home/niels/git/plutus-bench/contracts/always-succeeds/impl/plu-ts
npm ERR! command failed
npm ERR! command sh -c -- npm install @harmoniclabs/plu-ts

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/niels/.npm/_logs/2023-03-04T18_15_38_997Z-debug-0.log

Reply:
I assume the tagged version of plu-ts was not publicly released. Please make a public release before including the contract so that the build can be reproduced

Reply:
Hey I tried to answer from the desktop but apparently my previous answers are marked as "Pending", not sure why

I'm reporting one of the previous answers



Oh is probably due to the github registry requiring an access token in .npmrc;

here's some help
https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-npm-registry#installing-a-package

Reply:
I tried logging into the namespace @harmoniclabs and it didn't appear to work. I would also prefer a solution where such a setup (and especially requiring a GitHub account to build) is not required.

Reply:
Sure I'll update the PR as soon the package is published to the non registry then